commit 6a9b4b5b5ff302bebfca05a482b3c38a5c1d45b2 (refs/changes/92/1792/3)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-04-27T21:49:03+00:00 (2 years, 5 months ago)
    
    DOCKER-1027 /test/unit/helpers.js should be removed
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/test/unit/helpers.js b/test/unit/helpers.js
deleted file mode 100644
index af73f46..0000000
--- a/test/unit/helpers.js
+++ /dev/null
@@ -1,23 +0,0 @@
-/*
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-/*
- * Copyright (c) 2015, Joyent, Inc.
- */
-
-/*
- * Test helpers for SDC Docker unit tests
- */
-
-var common = require('../lib/common');
-
-
-
-// --- Exports
-
-module.exports = {
-    ifErr: common.ifErr
-};
